.github/workflows/e2e.compatibility.yaml

```diff
@@ -23,93 +23,40 @@ on:
 
 env:
   ISTIO_VERSION: 1.7.1
-  SCRIPTS_DIR: main/test/e2e-mesh/e2e-istio/scripts
-  SW_OAP_BASE_IMAGE: openjdk:8-jre-alpine
+  SATELLITE_TAG: v4b97fd255b6459fe9d3fc4f88c0d8f682fc2a364
+  SATELLITE_REPO: ghcr.io/apache/skywalking-satellite/skywalking-satellite
```

Comment:
Move these into the e2e environment file

Reply:
From here, I think repo path should be moved too.

Reply:
Seems this comment is not resolved?